backend/danswer/background/celery/tasks/indexing/tasks.py

```diff
@@ -64,7 +64,10 @@ def check_for_indexing(tenant_id: str | None) -> int | None:
     try:
         # these tasks should never overlap
         if not lock_beat.acquire(blocking=False):
+            task_logger.info(f"Lock acquired for tenant (Y): {tenant_id}")
```

Comment:
Are the Y and N reversed?  Also, seems easier to just write in english "Lock acquired" or "Lock not acquired".  Also the amount of logging for locks will be spammy, I'm not sure this logging should be INFO level.

Reply:
Good catch! Did not intend to leave these in- very spammy